cql/ManageCommonAbnormality.cql

```diff
@@ -504,10 +508,23 @@ define Action:
           TreatmentRecommendationText
         }
       }
-     when (
+    when (
       ErrataRecommendation is not null
     ) then
       ErrataRecommendation
+    when ( // Handle Special Case < 25 NILM & hrHPV positive
+      AgeInYears() < 25 and
+      MostRecentCytologyCotestWasAfterAge21 and
+      Dash.MostRecentNilmCytology is not null and
+      Dash.HasRecentPositiveHpv
```

Comment:
Does the positive HPV have to be a part of the same cotest as the NILM cytology test?

Reply:
Yes, this applies to the most recent cotest with NILM/HPV+

Reply:
In that case, I don't see anything associating the two tests on lines 518 and 519 together here. Should we consider using something like (MostRecentNegativeCotest)[https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/main/cql/ScreeningAverageRiskLibrary.cql#L160]?

Reply:
We could create a new function to look for both as part of a cotest if required.

Reply:
I thought you were saying above that the two tests being a part of a cotest _is_ required? I think this would be worth doing in this PR, but open to thoughts if you think otherwise.

Reply:
I pushed https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/101/commits/ccf1d773759c997295e2161ea85df0e07e6145b8 which should address this. @mrnosal can you review? Once you give the okay, for this change, I'll approve the PR.

Reply:
Mike gave the okay over Slack. Closing this thread as this has been resolved.